CHANGELOG-V2.md

```diff
@@ -3,6 +3,42 @@
 This document tracks changes to Temporal and its related projects for all `v2.x.x`
 releases. See our [versioning policy](/VERSIONING.md) for more details.
 
+## v2.0.4
+
+* relevant PRs:
+  * [#305](https://github.com/RTradeLtd/Temporal/pull/305)
+  * [#306](https://github.com/RTradeLtd/Temporal/pull/306)
+  * [#307](https://github.com/RTradeLtd/Temporal/pull/307)
+  * [#308](https://github.com/RTradeLtd/Temporal/pull/308)
```

Comment:
🤔 this is fine i guess but in the future we should try to make sure each fix gets its own PR, so that people can better see changes for themselves and vet them if they want

Reply:
this is a good idea, will do this from now on :ok_hand: